targetpos: [[156,477,0.0], [230, 404,1.5], [297,324,0.0], [419,185,0.0]]
time_out: 600
auto_mode: 0
